### PR TITLE
Build example link and formatting fixes to README.md

### PR DESCRIPTION
**Pull request by mlavi**
Hello, I wanted to add a helpful build link I found and then I cleaned up a lot of formatting for consistent capitalization, etc.
I added a Wikipedia reference to `doas` ~~and build artifacts to .gitignore~~. I hope you find this helpful!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,8 @@ version.h
 
 config.mk
 config.h
+
+description-pak
+doc-pak/
+*.deb
+
```
Comment (ericonr) [inline .gitignore:19]: I don't think it makes sense to add gitignore entries for artifacts generated by other software that's not the build system.

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ fraction of the codebase.
 There are a few steps you have to carefully consider before building and installing
 OpenDoas:
 
-* There are less eyes on random doas ports, just because `sudo` had a vulnerability
+* There are less eyes on random `doas` ports, just because `sudo` had a vulnerability
```
Comment (ericonr) [inline README.md:13]: "fewer eyes" ?

File: README.md [hunk 3]
```diff
@@ -53,9 +53,9 @@ file.
 ### Persist/Timestamp/Timeout
 
 The persist feature is disabled by default and can be enabled with the configure
-flag `--with-timestamp`.
+flag: `--with-timestamp`.
```
Comment (ericonr) [inline README.md:56]: "with the `--with-timestamp` configure flag" ?

File: README.md (hunk 2 above)
Comment (mlavi) [reply README.md:13]: Verbiage carried forward from previous author.

Comment (mlavi) [reply README.md:13]: Sure, I can incorporate that suggestion.

File: README.md (hunk 3 above)
Comment (mlavi) [reply README.md:56]: Incorporated.

File: .gitignore (hunk 1 above)
Comment (mlavi) [reply .gitignore]: Understood, removed with today's commit.

**Comment by mlavi** [comment]
The content is good, but I made two commit message typos, forgive me. :)

**Comment by mlavi** [comment]
@ericonr Anything else I need to do? Was it bad form to respond to each and then mark resolved, or should I have let you review and resolve?

File: README.md [hunk 4]
```diff
@@ -1,63 +1,66 @@
...
 * Use the default make target.
-* If you really want to install a setuid binary that depends on
-  pam being correctly configured, use the make install target
-  to install the software.
+  * If you really want to install a setuid binary that depends on
+    PAM being correctly configured, use the `make install` target
+    to install the software.
 
-## About the port
+*Caveat emptor:* a [build and installation](https://axebase.net/blog/2020/10/28/doas-das-kleine-sudo/) example for Debian/Ubuntu derivatives (German).
```
Comment (Duncaen) [inline README.md:25]: I removed the previous build instructions and I prefer to not include them. I don't want to have to deal with support requests about pam. There are distribution packages available for opendoas and they should be used. If they are not available, building software with configure and make should be easy enough for users who really want to configure pam and build opendoas themselves.

**Comment by ericonr** [comment]
> Was it bad form to respond to each and then mark resolved, or should I have let you review and resolve?

No worries about doing that :)

I'd recommend re-organizing the commits into more logical changes as well.

File: README.md (hunk 4 above)
Comment (mlavi) [reply README.md:25]: Fair enough, I'll rescind that and work on the squash commit @ericonr suggested!

**Pull request closed without merging**